apps/docs/src/styles/_kv-editor.scss

```
@@ -95,6 +95,39 @@
+      margin-inline: auto;
+      pointer-events: none;
+
+      &::before {
+        content: '';
+        position: absolute;
+        right: -5.6rem;
+        bottom: -3.125rem; // エディター上端（この行の 3.5rem 下）の少し上に矢印の先端が乗る位置
+        width: 10rem;
+        aspect-ratio: 426 / 281; // 書き出し PNG の実寸比
+        background-color: var(--gray-hi-c);
+        $editHere: url('/img/kv-edit-here.png');
```

Stylelint (17.14.0)

[error] Expected an empty line before $-variable (scss/dollar-variable-empty-line-before)

(scss/dollar-variable-empty-line-before)